components/esp_hid/src/ble_hidd.c

```diff
@@ -331,8 +332,15 @@ static esp_err_t create_hid_db(esp_ble_hidd_dev_t *dev, int device_index)
                 report->index = index;
                 add_db_record(_last_db, index++, (uint8_t *)&s_hid_report_uuid, ESP_GATT_PERM_READ, report->value_len, 0, NULL);
                 add_db_record(_last_db, index++, (uint8_t *)&s_character_client_config_uuid, ESP_GATT_PERM_READ | ESP_GATT_PERM_WRITE, 2, 0, NULL);
-            } else {
-                //Output or Feature Report
+            } 
+	    else if (report->report_type == ESP_HID_REPORT_TYPE_OUTPUT) {
```

Comment:
could you please make sure that spacing and styling is consistent with the rest of the file?